File: DataScienseProject/Models/MainPageModel.cs
using System.Collections.Generic;

namespace DataScienseProject.Models
{
    public class MainPageModel
    {
        public List<ProjectTypeModel> ProjectTypeModels { get; set; }
        public List<ExecutorModel> ExecutorModels { get; set; }
        public List<TechnologyModel> TehnologyModels { get; set; }
        public List<LayoutDataModel> LayoutDataModels { get; set; }
    }
}

File: DataScienseProject/ClientApp/src/app/models/main-page.model.ts
import { ExecutorModel } from "./executor.model"
import { LayoutDataModel } from "./layout-data.model"
import { ProjectTypeModel } from "./project-type.model";
import { TechnologyModel } from "./technology.model";

export class MainPageModel {
  projectTypeModels: Array<ProjectTypeModel>;
  executorModels: Array<ExecutorModel>;
  tehnologyModels: Array<TechnologyModel>;
  layoutDataModels: Array<LayoutDataModel>;
}
